objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json

```
@@ -0,0 +1,35 @@
+{
+    "id": "official.terrain_surface.grid_blue",
+    "authors": ["73"],
+    "version": "1.0",
+    "originalId": "0000008B|#OR2GRDB|00000000",
+    "sourceGame": ["official"],
+    "objectType": "terrain_surface",
+    "properties": {
+        "colour": "light_blue",
+        "mapColours": [138, 10],
+        "price": 110
+    },
+    "images": 
+	[ 
+	 { "path": "images/icon.png", "format": "raw", "x": 2, "y": 2 },
+	 "$G1[2257..2313]"
+    ],
```

Keep the style consistent:

```suggestion
    "images": [{
            "path": "images/icon.png",
            "format": "raw",
            "x": 2,
            "y": 2
        },
        "$G1[2257..2313]"
    ],
```